### PR TITLE
feat: almost done with landing page of e-commerce

### PR DESCRIPTION
**Pull request by dev-prabhat**

### CODE REVIEW

**Review by zoef-yt — APPROVED** [review]
Looks good to me. Good to merge.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```
Comment (Shubhamkr2610) [inline style.css]: I think you should give units here.

**Review by Shubhamkr2610 — APPROVED** [review]
Looks good to me. You can merge.

**Review by Muzammil-khan710 — APPROVED** [review]
Looking great.. good to merge

**Pull request merged**